sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeSentimentTests.cs

```diff
@@ -426,7 +426,6 @@ private void CheckAnalyzeSentimentProperties(DocumentSentiment doc, bool opinion
                 Assert.IsNotNull(sentence.ConfidenceScores.Neutral);
                 Assert.IsNotNull(sentence.ConfidenceScores.Negative);
                 Assert.IsTrue(CheckTotalConfidenceScoreValue(sentence.ConfidenceScores));
-                Assert.Greater(sentence.Length, 0);
```

Comment:
why deleting this?

Reply:
now that the property is gone, this doesn't provide value.
It is true that I could validate that the Text property doesn't return `string.empty` . We have an issue for that => https://github.com/Azure/azure-sdk-for-net/issues/14833

Reply:
never mind, i was confused that it is for a list not a string.